### ISSUE TITLE
Cropping image when exporting

### ISSUE THREAD

**Issue by xenongattz** [avocode/avocode#408]
I faced to one problem, that visible on screenshot.
The problem is that I can't crop image when exporting (see preview image and what is in main window).
In resulting design it's not visible, because bottom part of background image is under menu. But when exporting, it's shown.
So it's good to have option to crop image when I exporting image to file.

![2015-08-27 13 09 27](https://cloud.githubusercontent.com/assets/6165014/9518111/eba65c08-4cbc-11e5-978a-330895ddabb7.png)

**Comment by petrbrzek** [comment]
Good idea. This feature would be quite handy. We need to a good UI for it.

**Comment by tairov** [comment]
Any news here? ))
The feature is really needed. When I try use slices for this, it exports an image with the background. So, having an ability to export an image with cropping would be very helpful.
